files/zh-cn/web/api/barcode_detection_api/index.md

```diff
@@ -0,0 +1,259 @@
+      </td>
+      <td>
+        <img
+          alt="一个 code 93 条形码的图像。一个垂直黑白线的水平布局图案"
+          src="code-93.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>codabar</td>
+      <td>
+        一种表示字符 0-9、A-D 和符号 - $ / + 的线性条形码。
+      </td>
+      <td>
+        <img
+          alt="一个 codabar 格式条形码的图像。一个黑白垂直线的水平布局团"
+          src="codabar.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>data_matrix</td>
+      <td>
+        一种与方向无关的二维码，由按照 ISO16022 标准排列成正方形或矩形图案的黑白模块组成。
+      </td>
+      <td>
+        <img
+          alt="一个数据矩阵条形码的示例。一个由较小的黑色和白色方块填充的正方形"
+          src="data-matrix.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>ean_13</td>
+      <td>
+        一种基于 UPC-A 标准并在 iso15420 中定义的线性条形码。
+      </td>
+      <td>
+        <img
+          alt="一个 EAN-13 格式条形码的图像。一个白色和黑色线条的水平布局的图案"
+          src="ean-13.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>ean_8</td>
+      <td>一种在 ISO15420 中定义并源自 EAN-13 的线性条形码。</td>
+      <td>
+        <img
+          alt="一个 EAN-8 格式条形码的图像。一个垂直黑白线的水平布局的图案"
+          src="ean-8.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>itf</td>
+      <td>
+        一种连续、自检、双向可解码的条形码。它始终编码 14 位数字。
+      </td>
+      <td>
+        <img
+          alt="一个 ITF 条形码的图像。一个白色和黑色线条的水平布局图案"
+          src="ift.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>pdf417</td>
+      <td>
+        一种具有多行和多列的连续二维条形码符号格式。它可双向解码并且使用 iso15438 标准。
+      </td>
+      <td>
+        <img
+          alt="一个 pdf417 条形码格式的示例。一个由较小的黑白正方形组成的矩形"
+          src="pdf417.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>qr_code</td>
+      <td>
+        一种采用 iso18004 标准的二维条形码。编码的信息可以是文本、URL 或其他数据。
+      </td>
+      <td>
+        <img
+          alt="一个二维码的示例。一个由较小的黑色和白色正方形组成的正方形"
+          src="qr-code.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>upc_a</td>
+      <td>
+        最常见的线性条形码类型之一，广泛应用于美国的零售业。在 iso15420 中定义，它通过条带和空格来表示数字，每个数字都与 2 个条块和 2 个空格的独特模式相关联，两者的宽度都是可变的。UPC-A 可以编码唯一分配给每个项目的 12 位数字，从技术上讲，它是 EAN-13 的子集（UPC-A 代码在 EAN-13 中表示，第一个字符设置为 0）。
+      </td>
+      <td>
+        <img
+          alt="一个 upc-a 条形码的图像。一个由黑白垂直线组成且下方有数字的矩形"
+          src="upc-a.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>upc_e</td>
+      <td>
+        一种 iso15420 中定义的 UPC-A 的变体，压缩掉不必要的零以获得更紧凑的条形码。
+      </td>
+      <td>
+        <img
+          alt="一个 upc-e 条形码的图像。一个由黑白垂直线组成的矩形"
+          src="upc-e.png"
+        />
+      </td>
+    </tr>
+    <tr>
+      <td>unknown</td>
+      <td>
+        平台使用此值来表示它不知道或指定目前正在检测或支持哪种条形码格式。
+      </td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+你可以通过 {{domxref('BarcodeDetector/getSupportedFormats_static','getSupportedFormats()')}} 方法检查用户代理支持识别的格式。
+
+## 接口
+
+- {{domxref("BarcodeDetector")}} {{Experimental_Inline}}
+  - : Barcode Detection API 的 **`BarcodeDetector`** 接口允许检测图像中的条形码和二维码。
```

Comment:
We could remove the bold style, cloud you fix the English documentation?

```suggestion
  - : Barcode Detection API 的 `BarcodeDetector` 接口允许检测图像中的条形码和二维码。
```

Reply:
see https://github.com/mdn/content/pull/30976